### PR TITLE
Rename generator.rb constant

### PR DESCRIPTION
**Pull request by ghost**
This is a PR  in response to issue #11 as a part of my contribution towards hacktoberfest2019. Please do let me know how can I improve?

### CODE REVIEW

**Comment by pipimumu** [comment]
This is a PR  in response to issue #11 as a part of my contribution towards hacktoberfest2019. Please do let me know how can I improve?

**Comment by bronzdoc** [comment]
Thanks for the PR @pipimumu!

**Review by bronzdoc — APPROVED** [review]

File: lib/afip_bill/generator.rb [hunk 1]
```diff
@@ -11,7 +11,7 @@ class Generator
 
     HEADER_PATH = File.dirname(__FILE__) + '/views/shared/_factura_header.html.erb'.freeze
     FOOTER_PATH = File.dirname(__FILE__) + '/views/shared/_factura_footer.html.erb'.freeze
-    BRAVO_CBTE_TIPO = { "01" => "Factura A", "06" => "Factura B" }.freeze
+    CBTE_TIPO = { "01" => "Factura A", "06" => "Factura B" }.freeze
```
Comment (lubc) [inline lib/afip_bill/generator.rb]: @pipimumu Please update the name on the constant in the `type_a_or_b_bill` method as well

**Review by lubc — COMMENTED** [review]
@pipimumu Just added one comment. Also, there's a git conflict to address.

Thanks!

File: lib/afip_bill/generator.rb (hunk 1 above)
Comment (ghost) [reply lib/afip_bill/generator.rb]: @lubc  I have made  the aforesaid changes. Please do have a look!

**Comment by lubc** [comment]
@pipimumu Looks good! I can merge this once you fix the git conflict.

**Comment by ghost** [comment]
Hopefully this time we won't face any problems to merge this PR

File: lib/afip_bill/generator.rb [hunk 2]
```diff
@@ -11,9 +11,10 @@ class Generator
 
     HEADER_PATH = File.dirname(__FILE__) + '/views/shared/_factura_header.html.erb'.freeze
     FOOTER_PATH = File.dirname(__FILE__) + '/views/shared/_factura_footer.html.erb'.freeze
-    BRAVO_CBTE_TIPO = { "01" => "Factura A", "06" => "Factura B", "11" => "Factura C" }.freeze
+    CBTE_TIPO = { "01" => "Factura A", "06" => "Factura B" }.freeze
```
Comment (lubc) [inline lib/afip_bill/generator.rb]: @pipimumu The change that was on master has to remain. So this should be:

```
CBTE_TIPO = { "01" => "Factura A", "06" => "Factura B", "11" => "Factura C" }.freeze
```

Comment (ghost) [reply lib/afip_bill/generator.rb]: Please do have a look now @lubc Sir!

**Comment by lubc** [comment]
@pipimumu It looks great now 💯 Thanks!

**Pull request merged**